Wizualizacja gry/main.cpp

```diff
@@ -0,0 +1,78 @@
+
+//
+// Disclaimer:
+// ----------
+//
+// This code will work only if you selected window, graphics and audio.
+//
+// Note that the "Run Script" build phase will copy the required frameworks
+// or dylibs to your application bundle so you can execute it on any OS X
+// computer.
+//
+// Your resource files (images, sounds, fonts, ...) are also copied to your
+// application bundle. To get the path to these resources, use the helper
+// function `resourcePath()` from ResourcePath.hpp
+//
+
+//SFML Libraries
+#include <SFML/Graphics.hpp>
+
+//My Libraries
+#include "TankClass.hpp"
+#include "ObstacleClass.hpp"
+
+//Helping Libraries For SFML ( MacOS )
+#include "ResourcePath.hpp"
+
+int main(int, char const**)
+{
+    sf::RenderWindow window(sf::VideoMode(800, 600), "TankBotFight");
+
+    window.setFramerateLimit(1);
+    
+    Tank tankA(400, 300);
+    
+    Obstacle obstracleA(100, 100);
+    
+    while (window.isOpen())
+    {
+        
+        sf::Event event;
+        while (window.pollEvent(event))
+        {
+            
+            if (event.type == sf::Event::Closed) {
+                window.close();
+            }
+            
+            if (event.type == sf::Event::KeyPressed && event.key.code == sf::Keyboard::Escape) {
+                window.close();
+            }
+        }
+        
+        /*
```

Comment:
Let's not keep unused comments in repository

Reply:
This code will be used, but in another form. I had to test if it move accordingly. In the future this will be getting information in which direction to move and move an object.

Reply:
I believe this snippet is so simple that we don't have to store it in the repo. I would say keeping commented code in the repository is usually a bad practice. It's not adding any value to this file and just decrease the readability.